logstash-input-file.gemspec

```diff
@@ -28,7 +28,8 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'logstash-codec-multiline', ['~> 2.0.7']
 
   s.add_development_dependency 'stud', ['~> 0.0.19']
-  s.add_development_dependency 'logstash-devutils'
+  s.add_development_dependency 'logstash-devutils', ['~> 0.0.18']
```

Comment:
@purbon - Travis and Jenkins suddenly started using v0.0.8 see https://github.com/logstash-plugins/logstash-input-file/commit/68d3d6659e54ec331d1d2d64601a02db1e47a3cd and [Jenkins console](http://build-eu-00.elastic.co/job/logstash-plugin-input-file_PR/23/console)